docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,80 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* Improve client protocol throughput by introducing the spooling protocol.
```

Comment:
@wendigo .. once PR is merged I will link .. but otherwise.. good?

Reply:
If we're introducing a new protocol, we should probably say add, not improve.
``Add support for spooling client protocol, which...``

Reply:
Its technically an extension and fully compatible so its the same protocol still..

Reply:
Also the user impact is mostly the throughput improvement.. it really only make sense fully if we merge the docs PR.

Reply:
it's new protocol since it requires new clients for users to be able to make use of it.

it's forward compatible though (i.e. old clients can still access new servers)